modules/shell/command_test.go

```diff
@@ -89,6 +90,7 @@ wait
 	cmd := Command{
 		Command: "bash",
 		Args:    []string{"-c", bashCode},
+		Logger:     logger.With(logger.Discard),
```

Comment:
```suggestion
		Logger:     logger.New(logger.Discard),
```

Reply:
not quite, as `logger.Discard` is of type `*Logger, we can just use it directly even 🙂